modules/form/main/index.scss

```diff
@@ -11,17 +11,26 @@
       display: table-cell;
     }
 
-    > *:not(input) {
+    > *:not(input):not(select):not(textarea):not(button):not(.hx-btn) {
```

Comment:
Would {code}:not(input, select, textarea, button, .hx-btn){code} work ? It might be more readable ?